release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2744,7 +2744,7 @@ In the following tables, features are marked with the following statuses:
 |Cert-manager Operator
```

Comment:
```suggestion
|cert-manager Operator
```
https://cert-manager.io/docs/faq/#how-to-write-cert-manager : " cert-manager should always be written in lowercase. Even when it would normally be capitalized such as in titles or at the start of sentences. A hyphen should always be used between the words, don't replace it with a space and don't remove it. "

Reply:
@xingxingxia Addressed